.dockerignore

```diff
@@ -0,0 +1 @@
+docker-db-data
```

Comment:
This commit c3108bd54d009357d5d7d8e76f9655be89ee54f2 doesn't really belong here...

Reply:
I know. This PR is still in draft :pray: